app/views/api/clidocs/casestudy_metagenomics.html.erb

```diff
@@ -422,6 +422,35 @@
         </p>
       </div>
     </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Recap</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          The complete UMGAP can be written as a single command.
+          This allows the steps to run in parallel, further speeding up the final result.
+        </p>
+        <pre>
+<!--   --><b>$</b> umgap fastq2fasta A1.fq A2.fq |          # joining paired-end files
+<!--          -->  umgap translate -a |                     # translating all six frames
+<!--          -->  umgap prot2kmer2lca -o -k9 9-mer.index | # mapping each 9-mer onto a taxon or 0
+<!--          -->  umgap seedextend |                       # filtering extended seeds
+<!--          -->  sed '/^>/s_/.*__' |                      # dropping the paired-end and frame annotations
+<!--          -->  umgap uniq |                             # joining equal headers (all taxa of the same read)
+<!--          -->  umgap taxa2agg taxons.tsv |              # aggregating all taxa
+<!--          -->  grep -v '^>' |                           # dropping headers
+<!--          -->  umgap report taxons.tsv                  # make frequency table
+<!--   -->1,332163,Candidatus Solibacter usitatus
+<!--   -->1,392734,Terriglobus roseus
+<!--   -->1,940615,Granulicella tundricola
+        </pre>
+        <p>
+          This command processes the complete Metabenchmark A1 dataset on our computational server
+          (16 Intel Xeon CPU E5-2650 v2 at 2.60GHz, 20*16GB RDIMM x4 Data Width at 1866MHz, with the 9-mer index loaded in RAM)
```

Comment:
The Adler VM only has 8 cores and 200GB ram of the total amount.

_[Original comment](https://github.ugent.be/unipept/unipept/pull/649#discussion_r16591) by @bmesuere on Thu Aug 23 2018 at 15:04._